### DISCUSSION TITLE
Testing Uploads

### DISCUSSION THREAD

**Discussion by victorlevasseur** [Netflix/dgs-framework#209] [Q&A]
Hello,

We successfully made a mutation using the Upload scalar and MultipartFile with DGS. However, we are now trying to test it using the DgsQueryExecutor (and the codegen) but we are unable to find a way to pass the file to query.

How can it be done ?

**Comment by srinivasankavitha** [comment]
We have an example here: 
https://github.com/Netflix/dgs-framework/blob/master/graphql-dgs-example-java/src/test/java/com/netflix/graphql/dgs/example/FileUploadMutationTest.java

Hope this helps.
